src/Transports.Subscriptions.WebSockets/WebSocketConnection.cs

```diff
@@ -18,9 +18,9 @@ public class WebSocketConnection
 
         public async Task Connect()
         {
-            await _server.OnConnect();
-            await _server.OnDisconnect();
-            await _transport.CloseAsync();
+            await _server.OnConnect().ConfigureAwait(false); ;
+            await _server.OnDisconnect().ConfigureAwait(false); ;
```

Comment:
Please remove duplicated semicolons here and there.

Reply:
@sungam3r done